recipe/meta.yaml

```diff
@@ -1,22 +1,18 @@
 {% set name = "langdetect" %}
-{% set version = "1.0.7" %}
-{% set bundle = "zip" %}
-{% set hash_type = "sha256" %}
-{% set hash = "91a170d5f0ade380db809b3ba67f08e95fe6c6c8641f96d67a51ff7e98a9bf30" %}
-{% set build = 0 %}
+{% set version = "1.0.8" %}
```

Comment:
Thanks! There are still quite a few of those around and the maintainer abandoned them. Consider adding yourself as a maintainer in those cases.